pkg/controller/elasticsearch/driver/driver.go

```diff
@@ -140,6 +141,9 @@ func (d *defaultDriver) Reconcile(ctx context.Context) *reconciler.Results {
 
 	externalService, err := common.ReconcileService(ctx, d.Client, services.NewExternalService(d.ES), &d.ES)
 	if err != nil {
+		if apierrors.IsAlreadyExists(err) {
+			return results.WithReconciliationState(defaultRequeue.WithReason("Could not reconcile external service"))
```

Comment:
Open to any suggestions for this message that is used in the `ReconciliationComplete` status condition:

```yaml
> k get es c0 -o yaml | yq .status
availableNodes: 1
conditions:
  - lastTransitionTime: "2024-10-30T11:40:06Z"
    message: Could not reconcile external service  # 👈 👈 👈 👈 👈
    status: "False"
    type: ReconciliationComplete
  - lastTransitionTime: "2024-10-30T11:36:02Z"
    message: All nodes are running version 8.15.3
    status: "True"
    type: RunningDesiredVersion
  - lastTransitionTime: "2024-10-30T11:39:49Z"
    message: Service default/c0-es-internal-http has endpoints
    status: "True"
    type: ElasticsearchIsReachable
  - lastTransitionTime: "2024-10-30T11:38:57Z"
    message: 'Cannot get compute and storage resources from Elasticsearch resource generation 10: cannot compute resources for NodeSet "default": no CPU request or limit set'
    status: "False"
    type: ResourcesAwareManagement
health: unknown
inProgressOperations:
  downscale:
    lastUpdatedTime: "2024-10-30T11:35:31Z"
  upgrade:
    lastUpdatedTime: "2024-10-30T11:35:31Z"
  upscale:
    lastUpdatedTime: "2024-10-30T11:35:31Z"
observedGeneration: 11
phase: ApplyingChanges
version: 8.15.3
```